docs/tools/all.template_explanation.md

```diff
@@ -10,46 +11,56 @@
 
 <!-- tocstop -->
 
-## Introduction and motivation
+## Introduction / Motivation
+
+- What is this about?
+- What problem or limitation does it solve?
+- Who needs to understand it?
 
-- What is tool about?
-- What problem or limitation does this tool solve?
-- Who needs to understand this tool?
+Example: "Traditional file conversion methods focused on quality rather than
+file size. This methods aims to optimize file size while ensuring quality.
 
 ## Core Concepts
 
-- List and explain key ideas, building blocks, or theoretical foundations.
-  - **Concept 1:** Brief explanation
-  - **Concept 2:** How it relates to others
-  - **Concept 3:** Dependencies or assumptions
-- Attach diagrams or charts (e.g., using Graphviz dot, Mermaid, Tikz) whenever
-  possible or necessary.
+List and explain key ideas, building blocks, or theoretical foundations.
```

Comment:
Let's always use bullet lists even for level 0, so you can revert this and the following